### PR TITLE
Adds required checks to project visibility "Apply for review" button.

### PR DESCRIPTION
**Pull request by parrish**
Adds checkboxes to verify that the user has reviewed policies, best practices, and user feedback forms before requesting project review.

Deployed at https://beta-review-check.pfe-preview.zooniverse.org

cc/ @trouille

### CODE REVIEW

**Comment by parrish** [comment]
Just a reminder that projects have to be public and live to be eligible for review.

visibility

**Comment by trouille** [comment]
cc/ @chrislintott @mrniaboc   (This is in response to the suggestion from our Citizen Science Alliance advisory board call and the suggestion to have this simple addition to the Visibility tab before being able to click 'Apply for Review'.)

File: app/pages/lab/visibility.cjsx [hunk 1]
```diff
@@ -62,15 +99,41 @@ module.exports = React.createClass
...
+              </label>
+
+              <label>
+                <input type="checkbox" ref="bestPracticesReviewed" onChange={@toggleCheckbox.bind this, 'bestPracticesReviewed'} />
+                I have reviewed the <a href="/lab-best-practices" target="_blank">best practices</a>
+              </label>
+
+              <label>
+                <input type="checkbox" ref="feedbackReviewed" onChange={@toggleCheckbox.bind this, 'feedbackReviewed'} />
+                I have reviewed the <a onClick={@showFeedbackForm}>review feedback form</a>
```
Comment (brian-c) [inline app/pages/lab/visibility.cjsx]: Can we make this a `<button>`? Minor pet peeve.

File: app/router.cjsx [hunk 2]
```diff
@@ -142,6 +142,7 @@ module.exports =
     <Route path="lab-how-to" component={require './pages/lab/how-to-page'} />
 
     <Route path="lab-best-practices" component={require './pages/lab/best-practices'}>
+      <IndexRoute component={require './pages/lab/best-practices/introduction'} />
```
Comment (brian-c) [inline app/router.cjsx]: Does this duplicate the ./introduction route? Maybe redirect instead?

File: css/lab.styl [hunk 3]
```diff
@@ -53,6 +53,14 @@
       background: MAIN_HIGHLIGHT
       color: white
 
+.lab-visibility
+  &__policy-reviews
+    border: none
+    padding: 0
+
+    label
+      display: block
```
Comment (brian-c) [inline css/lab.styl]: Why make this a block? Results in the click area extending beyond the text.

File: app/pages/lab/visibility.cjsx (hunk 1 above)
Comment (parrish) [inline app/pages/lab/visibility.cjsx]: Sure.  Any preference for styling?  `.minor-button` maybe looks a bit big?

screen shot 2016-07-01 at 10 42 53 am

File: app/router.cjsx (hunk 2 above)
Comment (parrish) [inline app/router.cjsx]: Yeah, I can set that up.

File: app/pages/lab/visibility.cjsx (hunk 1 above)
Comment (brian-c) [inline app/pages/lab/visibility.cjsx]: Maybe I'd leave it as text and add a little "?" button after it, probably `className="pill-button"`. Or else you could add a button style that mimics links.

**Comment by parrish** [comment]
Updated, thanks!

**Pull request merged**